types/clamscan/tslint.json

```diff
@@ -0,0 +1,13 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "dt-header": false,
+        "npm-naming": [
+            true,
+            {
+                "mode": "name-only"
```

Comment:
If this was needed (and it's not clear that it is), please suppress the individual line(s) so that future lint violations aren't ignored.